WordPress/gutenberg
# Add functions to customize the `Group` and `Ungroup` buttons in the block contextual menu

Closes #49675

**This PR does not pass checks yet and doesn't have correct inline documentation yet. It's an idea I want to run past everyone first before putting in the effort to make it merge-able. It does, however, function.**

## What?

Add functions to customize the `Group` and `Ungroup` buttons in the block contextual menu.

## Why?

Since you can change the default block used to group blocks with `setGroupingBlockName`, you should be able to update the corresponding label in the UI as well.

## How?

This PR adds 2 publicly available functions to `set` and `get` the label for the grouping and ungrouping action in the Block Contextual Menu, sets the default values, and uses the functions to display the actual label.

## Testing Instructions




1. Create a new script in your theme or plugin and use the following code:
```js
import {
	setGroupingBlockNameLabel,
	setUngroupingBlockNameLabel,
} from "@wordpress/blocks";
import domReady from "@wordpress/dom-ready";

domReady(function () {
	setGroupingBlockNameLabel("Nest selection in a group");
	setUngroupingBlockNameLabel("Ungroup selection");
});
```
2. Open the WordPress Block Editor
3. Add a paragraph of text.
4. Open the Contextual block menu and see `Wrap in columns` as the option instead of `Nest selection in a group`. Select the option.
5. Open the Contextual block men of the newly created `group` block, and see both options updated, `Group` to `Nest selection in a group` and  `Ungroup` to `Ungroup selection`

### Testing Instructions for Keyboard


## Screenshots or screencast 
<img width="402" alt="screenshot showing the contextual menu with updated group and ungroup labels" src="https://user-images.githubusercontent.com/6925260/230777283-3b019152-c34c-439d-84e5-00c5ac874d2e.png">
